src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -180,14 +180,17 @@ def process_list(
             echo.echo_report(f'Using {percent_load * 100:.0f}% of the available daemon worker slots.')
 
 
-@verdi_process.command('show')
+@verdi_process.command('show', short_help='Show details for the given process(es).')
 @arguments.PROCESSES()
 @options.MOST_RECENT_NODE()
 @decorators.with_dbenv()
 def process_show(processes, most_recent_node):
-    """Show details for one or multiple processes."""
+    """Show details for one or multiple processes given the primary key(s) (PK)."""
     from aiida.cmdline.utils.common import get_node_info
 
+    if not (processes) and not (most_recent_node):
+        raise click.BadArgumentUsage('Please specify process primary key(s) (PK) or an option.')
```

Comment:
could maybe be more explicit about the problem and what option could be used?
```suggestion
        raise click.BadArgumentUsage(
            'No processes specified; specify one or more with arguments or use the `-M/--most-recent-node` flag.'
        )
```

Reply:
Agreed, I had the same comment